### DISCUSSION TITLE
Auto-backup

### DISCUSSION THREAD

**Discussion by nikitalogos** [stav121/i3wm-themer#133] [Ideas]
Hello. I used this application and it had just rewritten my .config file with no backup :) 
Writing "please backup your files first" at the end of your readme is not a good idea: most people would just skip it because of tldr. 
Please consider adding automatic backup feature, that will create .config.bk1 file or somehting like this if file already exists. Current behaviour is quite confusing(

**Comment by ryanalexmartin** [comment]
Current behavior is not only confusing, it's borderline malicious.

Why do you uninstall i3status?

Why do you overwrite so many things?  Literally everything I had on my system is broken and gone, and as the readme is written, `./install_ubuntu.sh` seems to be a dependency installer... I didn't expect it to automatically overwrite everything on my system.

This repo needs more than a disclaimer.  Very disappointed.
